src/Testcontainers.Spanner/Testcontainers.Spanner.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
+        <LangVersion>latest</LangVersion>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
+    <ItemGroup>
+        <PackageReference Include="JetBrains.Annotations" Version="2022.3.1"/>
+        <PackageReference Include="System.Net.Http.Json" Version="7.0.1" />
```

Comment:
I understand the point of that. For my implementation i considered this to be ok as in this case as it is a package owned by microsoft and it's .net framework team. I have no problem changing this to an implementation without this nuget package, just to make sure where you feel the boundary is: Do you consider any non framework package third party, even if it is owned by the framework creators?

Reply:
Yes, the package is generally fine. However, it may potentially force other developers to also use the specified version (7.0.1) in their own projects, leading to version conflicts. By avoiding "unnecessary" dependencies in modules, we can prevent developers from encountering this issue.